fe/fe-core/src/test/java/com/starrocks/sql/plan/LowCardinalityTest.java

```diff
@@ -1192,12 +1192,14 @@ public void testMultiMaxMin() throws Exception {
                 "min(upper(S_COMMENT)) from supplier_nullable " +
                 "group by upper(S_COMMENT)";
         plan = getFragmentPlan(sql);
-        Assert.assertTrue(plan.contains("  5:Decode\n" +
+        Assert.assertTrue(plan, plan.contains("6:Decode\n" +
```

Comment:
seems not as expected?

Reply:
The old
```
  6:Project
  |  <slot 11> : 11: max
  |  <slot 12> : 12
  |  
  5:Decode
  |  <dict id 17> : <string id 12>
  |  <dict id 15> : <string id 9>
  |  string functions:
  |  <function id 17> : DictExpr(14: S_COMMENT,[upper(<place-holder>)])
  |  <function id 15> : DictExpr(14: S_COMMENT,[upper(<place-holder>)])
  |  
  4:AGGREGATE (merge finalize)
  |  output: max(11: max), min(16: upper)
```

The new
```
6:Decode
  |  <dict id 17> : <string id 12>
  |  string functions:
  |  <function id 17> : DictExpr(14: S_COMMENT,[upper(<place-holder>)])
  |  
  5:Project
  |  <slot 11> : 11: max
  |  <slot 17> : 17: upper
  |  
  4:AGGREGATE (merge finalize)
  |  output: max(11: max), min(16: upper)
  |  group by: 15: upper
```
The new plan reduce a unnecessary col decode.